rc/extra/racer.kak

```diff
@@ -104,13 +104,20 @@ define-command racer-go-definition -docstring "Jump to where the rust identifier
         dir=${kak_opt_racer_tmp_dir}
         cursor="${kak_cursor_line} $((${kak_cursor_column} - 1))"
         racer_data=$(racer --interface tab-text  find-definition ${cursor} "${kak_buffile}" "${dir}/buf" | head -n 1)
+
         racer_match=$(printf %s\\n "$racer_data" | cut -f1 )
         if [ "$racer_match" = "MATCH" ]; then
           racer_line=$(printf %s\\n "$racer_data" | cut -f3 )
           racer_column=$(printf %s\\n "$racer_data" | cut -f4 )
           racer_file=$(printf %s\\n "$racer_data" | cut -f5 )
           printf %s\\n "edit -existing '$racer_file' $racer_line $racer_column"
-          #printf %s\\n "set-option buffer readonly true"
+          if [[ "$racer_file" == ${RUST_SRC_PATH}* ]]; then
```

Comment:
`[[` and `==` are bashisms, they're not portable that will work on arbitrary POSIX-compliant shells, you need to use:

`if [ "$foo" = "abc" ]; then`